app/api/search/specs/fixtures_elastic.js

```diff
@@ -166,9 +168,47 @@ export const fixtures = {
       published: true,
       user: userId,
     },
-    //metadata filters
+    //inherited aggregations
```

Comment:
There is not a single property inheriting non-id values: text properties, numbers, etc.  They should not get aggregated, but they should not break the app neither.  Perhaps, add one such property to ensure all is working as expected? (which the functional tests suggest this actually isn't working as expected)

Reply:
totally right.

Reply:
this has been fixed